code/lib/preview-api/src/modules/preview-web/PreviewWeb.test.ts

```diff
@@ -1552,6 +1552,32 @@ describe('PreviewWeb', () => {
       jest.useRealTimers();
     });
 
+    it('listens for events during setup', async () => {
```

Comment:
Maybe put this next to this one? it's a minor variant on it -- did you check it was failing before?

https://github.com/storybookjs/storybook/blob/428e6ce087c680c91c92a2eac74034069cb9dd72/code/lib/preview-api/src/modules/preview-web/PreviewWeb.test.ts#L1629

Reply:
I'm confused. No that story wasn't failing, but it seems to test for the very same thing.

Reply:
Slightly different in that the existing one didn't have a selection before init.